src/Engine/Graphics/Outdoor.cpp

```diff
@@ -2321,6 +2321,8 @@ void TeleportToStartingPoint(MapStartPoint point) {
             for (size_t i = 0; i < pLevelDecorations.size(); ++i) {
                 if (pLevelDecorations[i].uDecorationDescID == pDecorationList->GetDecorIdByName(pName)) {
                     pParty->pos = pLevelDecorations[i].vPosition;
+                    bool bOnWater = false;
+                    pParty->pos.z = GetTerrainHeightsAroundParty2(pParty->pos, &bOnWater, 0);
```

Comment:
Reading that bOnWater is useless.

Reply:
I know - it was just because I need to pass a pointer to bool and not a bool itself to the function